src/main/resources/hudson/plugins/sshslaves/Messages_es.properties

```diff
@@ -23,32 +23,32 @@
 # {0} [SSH] Remote file system root {1} does not exist. Will try to create it...
 SSHLauncher.RemoteFSDoesNotExist={0} [SSH] Directorio raiz en el sistema remoto ({1}) no existe. Intentando crearlo ...
 # {0} [SSH] Authentication successful.
-SSHLauncher.AuthenticationSuccessful={0} [SSH] Autenticaci�n v�lida
+SSHLauncher.AuthenticationSuccessful={0} [SSH] Autenticaci�n v�lida
```

Comment:
Looks like you corrupted this file.

Anyway you did not touch all the translations defining `SSHLauncher.NoJavaFound`. Perhaps better to just leave the translations alone, and define a new key.